### PR TITLE
Update platformdev_blackpill_f411.md

### PR DESCRIPTION
**Pull request by marfrit**
Add STM32F401 to headline to help future googlers not getting to the bootloader on those boards

## Description

Add the F401 type for "seo".

### CODE REVIEW

File: docs/platformdev_blackpill_f411.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# WeAct Blackpill (STM32F411)
+# WeAct Blackpill (STM32F411 & STM32F401)
```
Comment (tzarc) [inline docs/platformdev_blackpill_f411.md:1]:
```suggestion
# WeAct Blackpill (STM32F4x1)
```

**Review by tzarc — CHANGES_REQUESTED** [review]
Would be great if you could rename the file to be `...f4x1.md` as well.
There will be a couple of changes in the rest of the repo to get things like the sidebar to match up, a `grep` should help. Thanks!

File: docs/platformdev_blackpill_f411.md (hunk 1 above)
Comment (marfrit) [reply docs/platformdev_blackpill_f411.md]: Been there, done that. Added a sentence including F401 and F411 for searchers.

File: docs/platformdev_blackpill_f4x1.md [hunk 2]
```diff
@@ -1,8 +1,10 @@
-# WeAct Blackpill (STM32F411)
+# WeAct Blackpill (STM32F4x1)
+
+This document applies to the F411 and F401 based blackpills.
```
Comment (fauxpark) [inline docs/platformdev_blackpill_f4x1.md:3]:
```suggestion
This document applies to the F401- and F411-based Blackpills.
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**